examples/js/loaders/3MFLoader.js

```diff
@@ -1,3 +1,7 @@
+/**
+ * @author technohippy / https://github.com/technohippy
+ */
+
 THREE.ThreeMFLoader = function ( manager ) {
```

Comment:
What do you think about renaming it to `TMFLoader`? Just like `TDSLoader` is for 3DS...?